flink-table/flink-table-planner/pom.xml

```diff
@@ -196,14 +189,15 @@ under the License.
 			<!-- For legacy string expressions in Table API -->
 			<groupId>org.scala-lang.modules</groupId>
 			<artifactId>scala-parser-combinators_${scala.binary.version}</artifactId>
+			<scope>provided</scope>
 		</dependency>
 
 		<dependency>
 			<!-- Utility to scan classpaths -->
 			<groupId>org.reflections</groupId>
 			<artifactId>reflections</artifactId>
 			<version>0.9.10</version>
-			<scope>compile</scope>
+			<scope>provided</scope>
```

Comment:
Who pulls in this? not table-runtime.

Reply:
Check `ReflectionsUtil`. We can get rid of this dep once during the FLIP-190 related work.